docs/faststore/components/features/delivery-promise/overview.mdx

```diff
@@ -1,81 +1,127 @@
+* `useDeliveryPromise`: Manages all logic for querying, displaying, and updating delivery availability filters and states.
+
+## Filter options
+
+Delivery Promise offers shoppers two product filtering options:
+- [Global filters](#global-filters): Apply to all store navigation.
+- [Product List Page (PLP) or search page](#plp-or-search-page-filters): Apply only to the current page the shopper is viewing.
+
+>ℹ Learn more about filter usage and user flows on the [Delivery Promise Figma Community](https://www.figma.com/community/file/1545494767147168145/delivery-promise-by-vtex) page.
+
+
+### Global filters
+
+Global filters remain active across all user navigation and are applied to all product searches until they are changed or cleared.
+
+Shoppers can filter available products globally using the **Filter by store** option, which displays only products available for pickup at a specific store. This filter shows all available pickup points within a 50-km radius, as configured in Checkout. The number of pickup points displayed is unlimited.
+
+![filter-by-store](https://vtexhelp.vtexassets.com/assets/docs/src/dp-filter-by-store___6e22784e6d6a26719d8d7fbc6dfa0e43.gif)
+
+### PLP or search page filters
+
+On PLPs or search pages, the shopper has the following options for applying filters: [Delivery Methods](#delivery-methods-filters), [Delivery Option](#delivery-option-filters), and [Availability](#availability).
+
+
+>⚠️ To edit the filter labels, follow the instructions in [Managing filters and badges](https://developers.vtex.com/docs/guides/faststore/delivery-promise-implementation).
+
+#### Delivery Methods
+
+This filter displays available products according to the shopper's location. With this option, shoppers can filter using the following options:
+
+![dp-delivery-methods](https://vtexhelp.vtexassets.com/assets/docs/src/dp-filters-delivery-method___945ea013036a9ebebfc0078cb93796d9.png)
+
+| **Filter option** | **Description** |
+| ----------------- | --------------- |
+| **All delivery methods** | Displays products that match all active delivery method filters: **Shipping to**, **Pickup anywhere**, **Pickup at**, and **Pickup nearby**. |
+| **Shipping to**  |  Displays products that can be shipped to the shopper's specified location. |
+| **Pickup anywhere** | Displays products available for pickup at any participating store within the merchant's network, rather than restricting results to a selected pickup location or geographic proximity. This is often used by B2B stores, and this filter is deactivated in Headless CMS by default. |
+| **Pickup at** | Displays the store specified in the global pickup point filter or the one closest to the shopper's location. To change the store set in `Pickup at {storeName}`, the shopper can click on it and select a new store in the slideover that opens. This slideover shows all available pickup points within a 50-km radius, as configured in Checkout. The number of pickup points displayed is unlimited. |

```

Comment:
@lariciamota is unlimited for us, because we are only displaying what is coming from API right? Is this information somehow important to know? (Is it because you can't set a limit?)

Reply:
https://vendemo.myvtex.com/admin/logistics#/config
<img width="1207" height="200" alt="Screenshot 2025-10-15 at 13 22 08" src="https://github.com/user-attachments/assets/26d8aa05-004f-4af3-84e7-d9045d465da9" />